Utilities/macserial/cccdump.py

```diff
@@ -1,96 +1,99 @@
+# pylint: disable=undefined-variable,import-error,too-many-boolean-expressions
```

Comment:
That code is for python 2 actually. Though maybe now it is ok to upgrade, IDA supports python 3 today.

Reply:
You mean the whole script, right? It should be Py3 compliant, despite I couldn't test it (`ida_bytes` library missing on my system). How can I check it? changed stuff is just `print` statements, `l` variable name changed to `length` and `is None` syntax where needed.
Is there anything else I can do to test it? 
Sorry but I never used that script and have no clue on how to properly test it 😅

Reply:
Leave as is, I guess I will fix it if I need it again.

Reply:
Oh sure. Can I mark this conversation as solved then?